.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
+        {
+          "files": ["**/*.test.js", "**/*.mock.js"],
+          "env": { "jest": true },
+          "plugins": ["jest"],
+          "extends": ["plugin:jest/recommended"],
+          "rules": { "jest/prefer-expect-assertions": "off" }
+        },
+        {
+            "files": ["**/*.cy.js", "**/commands.js"],
+            "env": { "cypress/globals": true },
+            "plugins": ["cypress"],
+            "extends": ["plugin:cypress/recommended"],
+            "rules": {
+              "cypress/no-unnecessary-waiting": "off",
+              "no-unused-vars": "off"
+            }
+        }
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
+        "no-unused-vars": "warn"
```

Comment:
It's best not to add this to the global configuration as we should keep this rule as error for normal non-testing code